internal/storage/storage.go

```diff
@@ -160,8 +167,26 @@ func (s *Storage) GetHostses() ([]Hosts, error) {
 }
 
 // cluster
-func (s *Storage) InsertCluster(name, uuid, description, topology string) error {
-	return s.write(InsertCluster, uuid, name, description, topology)
+func (s *Storage) InsertCluster(name, uuid, description, topology, deployType string) error {
+	return s.write(InsertCluster, uuid, name, description, topology, deployType)
+}
+
+func (s *Storage) CheckTypeFiledExist() (int, error) {
```

Comment:
This func is to determine if type exists, so whether a `bool` return value is easier to understand

Reply:
I see, and I'll fix this